terminal/output.go

```diff
@@ -103,6 +110,8 @@ func (terminal *Terminal) processInput(pty chan rune) {
 
 	var b rune
 
+	//	debug := ""
```

Comment:
this probably shouldn't be left behind

Reply:
Left here intentionally so that I can turn on stream dump without writing it over again.